backend/src/services/user/emailVerification.js

```diff
@@ -9,7 +9,7 @@ export const sendEmailVerificationLink = async(email, name, id) => {
 
     EmailVerificationTokenRepo.create({userId: id, token: hash});
     
-    const link = `${process.env.BASE_URL}api/auth/verify-mail?token=${verificationToken}&id=${id}`;
+    const link = `${process.env.BASE_URL}/verifyemail?token=${verificationToken}&id=${id}`;
```

Comment:
The link for verifying email threw an error previously because there was no /api attached to it. Please, can you address this

Reply:
based on the frontend dev complaint, he asked me to make the link sent in each mail to route to a page on the frontend where user can click a button to verify

Reply:
okay. noted